### DISCUSSION TITLE
access to assets

### DISCUSSION THREAD

**Discussion by igor-yusupov** [fzyzcjy/flutter_rust_bridge#1230] [Q&A] [answered]
How can I access the assets files from Rust? I know I can save these files to a temporary directory through path_provider, but can I access them immediately?

**Comment by fzyzcjy** [comment, marked as answer]
> can I access them immediately?

This is more like a general Flutter question. Indeed, if you can access them using C code or Java/ObjC code, then I guess there is a way to do so in Rust.
